src/components/application_manager/src/commands/mobile/add_sub_menu_request.cc

```diff
@@ -102,19 +102,28 @@ void AddSubMenuRequest::Run() {
                  [strings::image_type].asUInt();
   const mobile_apis::ImageType::eType type =
       static_cast<mobile_apis::ImageType::eType>(image_type);
-  const bool is_dynamic_image = mobile_apis::ImageType::DYNAMIC == type;
-
-  if (is_key_icon_exist && is_icon_path_valid && is_dynamic_image) {
-    (*message_)[strings::msg_params][strings::sub_menu_icon][strings::value] =
-        ImageFullPath((*message_)[strings::msg_params][strings::sub_menu_icon]
-                                 [strings::value].asString(),
-                      app,
-                      application_manager_);
+
+  if (is_key_icon_exist && is_icon_path_valid) {
     msg_params[strings::sub_menu_icon] =
         (*message_)[strings::msg_params][strings::sub_menu_icon];
+    const bool is_dynamic_image = mobile_apis::ImageType::DYNAMIC == type;
+    if (is_dynamic_image) {
+      msg_params[strings::sub_menu_icon][strings::value] =
+          ImageFullPath((*message_)[strings::msg_params][strings::sub_menu_icon]
+                                   [strings::value].asString(),
```

Comment:
@pold500 please check check_style.sh on you code, lloks like formating corrupted